internal/characters/jean/burst.go

```diff
@@ -57,11 +57,11 @@ func (c *char) Burst(p map[string]int) action.ActionInfo {
 	ai.Mult = burstEnter[c.TalentLvlBurst()]
 	// first enter is at frame 55
 	for i := 0; i < enter; i++ {
-		c.Core.QueueAttackWithSnap(ai, snap, combat.NewCircleHit(c.Core.Combat.Player(), 5), 55+i*delay)
+		c.Core.QueueAttackWithSnap(ai, snap, combat.NewCircleHit(c.Core.Combat.Player(), 6), 55+i*delay)
```

Comment:
should technically be true single target on the enemy that's going in and out, maybe assume it hits PrimaryTarget?